build-configuration/bom/pom.xml

```diff
@@ -153,6 +158,11 @@
                 <artifactId>infinispan-client-hotrod</artifactId>
                 <version>${version.infinispan}</version>
             </dependency>
+            <dependency>
+                <groupId>org.infinispan</groupId>
+                <artifactId>infinispan-client-hotrod-reactive</artifactId>
```

Comment:
I can't find this artifact anywhere!

Reply:
good catch, I change the dependency name and totally forgot about the bom